include/svnlook.php

```diff
@@ -812,7 +812,8 @@ function applyGeshi($path, $filename, $language, $rev, $peg = '', $return = fals
 		$this->geshi->set_tab_width($this->repConfig->getExpandTabsBy());
 
 		if ($highlight == 'file') {
-			$this->geshi->enable_line_numbers(GESHI_FANCY_LINE_NUMBERS);
+			$this->geshi->enable_line_numbers(GESHI_FANCY_LINE_NUMBERS,1);
+			$this->geshi->set_overall_id('line');
```

Comment:
Attention, this does not only apply to the line, but to the entire geshi rendering.

Reply:
![image](https://user-images.githubusercontent.com/4973182/152693627-aed341f3-904d-432a-b31f-a7efe561bf99.png)
Yes this applies to all lines...

Reply:
`line-` is too naive. It will apply to all elements of the Geshi rending, not just lines. Lets make it `geshi` just like the `set_overall_class()`

Reply:
Are you sure?? making geshi would mean if we have to get a url for say line number 110 we have to use anchor as `#geshi-110`. I am thinking line would be more intuitive. like `#line-110` knowing we are trying to jump to line 110. or should we use `l` instead, like done in blame.php ??  
Here is the url in case of blame `http://localhost/websvn/blame.php?repname=Test+Group+1.Common+Model&path=%2Ftrunk%2FCFiles%2Fasm08.c&rev=23#l110`

Reply:
Yes, I am sure. Look into `geshi.php` and usage of `overall_id`. It does *not only* apply to numbers. It is all or nothing. As far I can see `l110` is not possible because the template is `{id}-{number}`. The hyphen remains.

Reply:
Okay... Changed to `geshi`.